src/rpc/blockchain.cpp

```diff
@@ -1754,8 +1754,10 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
-            + HelpExampleRpc("getblockstats", "1000 '[\"minfeerate\",\"avgfeerate\"]'")
+                    HelpExampleCli("getblockstats", "'\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\"' '[\"minfeerate\",\"avgfeerate\"]'") +
```

Comment:
Using raw string literals will make it this more readable.
```suggestion
                    HelpExampleCli("getblockstats", R"('"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09"', '["minfeerate","avgfeerate"]')") +
```

Reply:
Nice, rebased with your suggestion